.github/workflows/conventional-pr-title-checker.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 # cancel redundant builds
 concurrency:
-  group: "title-checker-${{ github.ref }}"
+  group: "title-checker-${{ github.head_ref }}"
```

Comment:
So, if you do that, the merge queue will not work anymore. I think it's important for this file to be consistent. If we keep the push event above, then we need to use something here that is independent of pull request. And head ref is not
Or, we simply remove the push event above, if we can't find something that works for both

Reply:
I am using fall back values now, head_ref if available otherwise it uses ref

https://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-a-fallback-value